terraform/stack/alarm.tf

```diff
@@ -0,0 +1,23 @@
+//TODO does this do anything
```

Comment:
No. I think it's been a long time since it did. Arguably it should do something, but we're not going to tackle it here and might choose to do it differently if we did, so delete it.